internal/output/output.go

```diff
@@ -99,8 +99,9 @@ type Output struct {
 // SetImageAccessibleCheck sets the passed and result.message fields of the ImageAccessibleCheck to the given values.
 func (o *Output) SetImageAccessibleCheckFromError(err error) {
 	metadata := map[string]interface{}{
-		"code":  "builtin.image.accessible",
-		"title": "Image URL is accessible",
+		"code":        "builtin.image.accessible",
+		"title":       "Image URL is accessible",
+		"description": "The image URL is available and accessible on a public registry.",
```

Comment:
It doesn't have to be a public registry/repository. The underlying system just has to have access to it. For example, if you have a `~/.docker/config.json`, ec-cli will use that.

Reply:
@lcarva I've updated it to remove "...on a public registry" - open to more detailed suggestions, though.